src/platform/packages/shared/kbn-scout/README.md

```diff
@@ -154,10 +171,112 @@ test.beforeEach(async ({ pageObjects }) => {
 });
 ```
 
-5. **src/servers/**
+6. **src/servers/**
+
+Here we have logic to start Kibana and Elasticsearch servers using `kbn-test` functionality in Scout flavor. The instance of the `Config` class is passed to start servers for the specific deployment type. The `flags.ts` file contains server-related command-line flags and options. The `loadServersConfig` function not only returns a `kbn-test` compatible config instance, but also converts it to `ScoutServiceConfig` format and saves it on disk to `./scout/servers/local.json` in the Kibana root directory. Scout `config` fixture reads it and expose to UI tests.
+
+### Test Types and Directory Structure
+
+Scout supports two distinct types of tests: UI and API, each with their own directory structure and import patterns:
+
+#### Setting up Test Directory
+
+To get started with Scout testing for your plugin, you need to create the appropriate directory structure in your plugin's root directory:
+
+```
+your-plugin/
+├── test/
+│   └── scout/
+│       ├── ui/
+│       │   ├── playwright.config.ts
+│       │   ├── parallel.playwright.config.ts
+│       │   └── parallel_tests/               # Your UI test specs, that are run in parallel
+│       │   └── tests/                        # Your UI test specs, that are run sequentially
+│       └── api/
+│           ├── playwright.config.ts
+│           └── tests/                        # Your API test specs, that are run sequentially
+```
+
+#### UI Tests
+
+UI tests are designed for browser-based integration testing and provide access to browser fixtures like `page`, `pageObjects`, and `browserAuth`.
+
+**Test Imports for UI Testing:**
+
+```ts
+// For sequential UI tests
+import { test, expect } from '@kbn/scout';
+
+// For parallel UI tests that can be space-isolated
+import { spaceTest as test, expect } from '@kbn/scout';
+```
+
+**When to use each:**
+
+- **`spaceTest`**: Use for parallel tests that can be isolated by Kibana spaces, allowing faster execution
+- **`test`**: Use for sequential tests that cannot run in parallel
+
+**Example UI Test:**
+
+```ts
+import { spaceTest as test, expect } from '@kbn/scout';
+
+test('should display dashboard', async ({ pageObjects, page }) => {
+  await pageObjects.dashboard.goto();
+  await expect(page.testSubj.locator('dashboardLandingPage')).toBeVisible();
+});
+```
+
+#### API Tests
+
+API tests are designed for server-side testing and provide fixtures focused on API interactions without browser-related fixtures.
+
+**Test Import for API Testing:**
+
+```ts
+// For API integration tests (server-side only, no browser fixtures)
+import { apiTest as test, expect } from '@kbn/scout';
+```
+
+**Example API Test:**
+
+```ts
+import { apiTest, expect } from '@kbn/scout';
+
+apiTest('POST api/painless_lab/execute is disabled', async ({ kbnClient, log }) => {
```

Comment:
Here we'll likely want to make the `apiClient` available because it's used within the test body to make an API request:

```suggestion
apiTest('POST api/painless_lab/execute is disabled', async ({ apiClient, log }) => {
```